components/dropdown/dropdown-content-mixin.js

```diff
@@ -403,6 +414,14 @@ export const DropdownContentMixin = superclass => class extends LocalizeCoreElem
 		}
 	}
 
+	__disconnectResizeObserver() {
+		this.renderCount++;
+		if (this.el.offsetHeight !== 0 && this.__resizeObserver) {
```

Comment:
We are just checking height here as the div has a height of zero but a non-zero width. Not sure if it's worth adding another option for consumers to also check width since it's less common to have a zero width and non-zero height.

Reply:
Is it possible to leverage the entries that are passed into the observer?  The purpose of those entries is to allow handlers to run logic using the provided dimensions without causing layout (which calling `offsetHeight` will).